packages/angular/src/directives/control-value-accessors/text-value-accessor.ts

```diff
@@ -3,7 +3,7 @@ import { NG_VALUE_ACCESSOR } from '@angular/forms';
 import { ValueAccessor } from '@ionic/angular/common';
 
 @Directive({
```

Comment:
~ We should probably consider refactoring this class in the next major release. It isn't a `TextValueAccessor` if we are using it for `ion-range`. If we are going to group all value accessors with a shared event, we might want to consider a `IonInputValueAccessor` class instead. Changing the class name will be a breaking change since this class can be used for dependency injection in Angular apps. 

Non-blocking for this PR, but something to consider.

Reply:
I added a tech debt ticket for this 👍